packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -30,56 +22,51 @@ export function Service({
 }) {
   const { city, stateCode, country } = location;
   const locationProvided = Boolean(city && stateCode && country);
-  const isPocket = name === 'Pocket';
-  const isMonitor = name === 'Firefox Monitor';
-  const isLockwise = name === 'Firefox Lockwise';
-  const isSync = name === 'Firefox Sync';
-  const isFPN = name === 'Firefox Private Network';
+  let serviceLink, iconName;
 
-  const renderLink = isPocket || isMonitor || isLockwise || isSync || isFPN;
-  let serviceLink = '';
-  let iconSvg = <WebIcon data-testid="web-icon" />;
-
-  if (renderLink) {
-    if (isPocket) {
-      serviceLink = 'https://www.mozilla.org/en-US/firefox/pocket/';
-      iconSvg = <PocketIcon data-testid="pocket-icon" />;
-    } else if (isMonitor) {
-      serviceLink = 'https://monitor.firefox.com/';
-      iconSvg = <MonitorIcon data-testid="monitor-icon" />;
-    } else if (isLockwise) {
-      serviceLink = 'https://www.mozilla.org/en-US/firefox/lockwise/';
-      iconSvg = <LockwiseIcon data-testid="lockwise-icon" />;
-    } else if (isSync) {
-      serviceLink =
-        'https://support.mozilla.org/en-US/kb/how-do-i-set-sync-my-computer';
-      iconSvg = <SyncIcon data-testid="sync-icon" />;
-    } else if (isFPN) {
-      serviceLink = 'https://vpn.mozilla.com/';
-      iconSvg = <FPNIcon data-testid="fpn-icon" />;
-    }
-  } else {
-    const lowerName = name.toLowerCase();
-    if (deviceType === 'mobile') {
-      iconSvg = <MobileIcon data-testid="mobile-icon" />;
-    }
-    if (lowerName.includes('ipad')) {
-      iconSvg = <TabletIcon data-testid="tablet-icon" />;
-    } else if (deviceType === 'desktop') {
-      iconSvg = <DesktopIcon data-testid="desktop-icon" />;
-    }
+  switch (name) {
+    case "Pocket":
+      serviceLink = "https://www.mozilla.org/en-US/firefox/pocket/";
+      iconName = "pocket";
+      break;
+    case "Firefox Monitor":
+      serviceLink = "https://monitor.firefox.com/";
+      iconName = "monitor";
+      break;
+    case "Firefox Lockwise":
+      serviceLink = "https://www.mozilla.org/en-US/firefox/lockwise/";
+      iconName = "lockwise";
+      break;
+    case "Firefox Private Network":
+      serviceLink = "https://vpn.mozilla.com/";
+      iconName = "fpn";
+      break;
+    case "Firefox Sync":
+      serviceLink = "https://support.mozilla.org/en-US/kb/how-do-i-set-sync-my-computer";
+      iconName = "sync";
+      break;
+    default:
+      iconName = "web";
+      if (deviceType === "mobile") {
+        iconName = "mobile";
+      }
+      if (name.toLowerCase().includes("ipad")) {
+        iconName = "tablet";
+      } else if (deviceType === "desktop") {
+        iconName = "desktop";
```

Comment:
Did this file get reformatted? Looks like this and the above are using double quotes now

Reply:
Yeah looks like everything in this commit is using double quotes.

Reply:
Fixed that, should be much more legible diff now